api/src/certification/enrolment/domain/models/Subscription.js

```diff
@@ -8,7 +8,7 @@ import { validate } from '../../../shared/domain/validators/subscription-validat
 class Subscription {
   /**
    * @param {Object} params
-   * @param {number} params.certificationCandidateId - identifier of the certification candidate
+   * @param {number || null} params.certificationCandidateId - identifier of the certification candidate
```

Comment:
```suggestion
   * @param {number} params.[certificationCandidateId] - identifier of the certification candidate
```
**question:** cas de "non required" ?

Reply:
J'ai effectivement fait une faute de syntaxe mais les [] ne permettent que de type `undefined` et non null qui est une valeur utilisée dans la codebase. (Je ne type que l'existant sans chercher pour l'instant à voir les incohérences possibles)